### ISSUE TITLE
specify: related parts

### ISSUE THREAD

**Issue by MortenHofft** [gbif/model-material#66]
> Context: I'm trying to reconstruct a single specimen https://www.gbif.org/occurrence/657029235
which I believe is the same as dc96c974-1ed3-11e3-bfac-90b11c41863e in the data we have in this repo. And I think this is the specify version: https://ichthyology.specify.ku.edu/specify/view/collectionobject/34592/

https://ichthyology.specify.ku.edu/specify/view/collectionobject/34592/ links to tissue as another collection object [000008548](https://ichthyology.specify.ku.edu/specify/view/collectionobject/49739/)

I do not believe that is included. At least I do not see the word `Muscle` nor the GUID `5c74480a-4c30-4558-b4a2-e052c377a219` anywhere in the CSVs

Is that excluded on purpose?

**Comment by acbentley** [comment]
Yes, again this goes to incorporating my tissue collection.  KUIT 8548 is in the tissue collection as related to that voucher record.  The tissue and voucher collections are published as two separate collections to GBIF - https://www.gbif.org/dataset/8f79c802-a58c-447f-99aa-1d6a0790825a and https://www.gbif.org/dataset/56caf05f-1364-4f24-85f6-0c82520c2792 respectively.  I also have an observation collection that is not as yet published.
